scripts/local_testnet/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM rust:1.50.0 AS builder
+RUN apt-get update && apt-get install -y cmake
+
+RUN git clone https://github.com/sigp/lighthouse
+ARG PORTABLE
+ENV PORTABLE $PORTABLE
+RUN cd lighthouse && make
+RUN env
+RUN cd lighthouse && make install-lcli
+
+##################################################
+FROM debian:buster-slim
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  libssl-dev \
+  ca-certificates \
+  nodejs npm \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
+
+COPY --from=builder /usr/local/cargo/bin/lighthouse /usr/local/bin/lighthouse
+COPY --from=builder /usr/local/cargo/bin/lcli /usr/local/bin/lcli
```

Comment:
I think it would be awesome if we could use our existing docker images to speed up this build.  I believe we can just copy the binary directly from the image on the dockerhub repo like this:
```
COPY --from=sigp/lighthouse:latest-unstable /usr/local/bin/lighthouse /usr/local/bin/lighthouse
COPY --from=sigp/lcli:latest-unstable /usr/local/bin/lcli /usr/local/bin/lcli
```
We recently added an `lcli` docker build to our CI but haven't actually published it to dockerhub yet, but should in the next week or so.